client/me/application-password-item/index.jsx

```diff
@@ -49,7 +49,7 @@ class ApplicationPasswordsItem extends React.Component {
 					<h2 className="application-password-item__name">{ password.name }</h2>
 					<p className="application-password-item__generated">
 						{ this.props.translate( 'Generated on %s', {
-							args: this.props.moment( password.generated ).format( 'MMM DD, YYYY @ h:mm a' ),
+							args: this.props.moment( password.generated ).format( 'LL @ LT' ),
```

Comment:
Should it be `ll @ LT` instead, because `MMM` would display only the first three letters of the month?